db/migrate/20230503164837_create_likes.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+# A like belongs to a user and a post.
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.integer :postId
+
+      t.timestamps
+    end
+  end
```

Comment:
- [ ] Take a look at the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png). You should create the associations or foreign keys column using [references](https://edgeguides.rubyonrails.org/active_record_migrations.html#creating-associations-using-references) column type also available as `belongs_to`. Your code should look similar to this:

```ruby
  create_table :likes do |t|
    t.timestamps
    t.references :post, foreign_key: true
  end

```
- This will create post_id and a corresponding index but students can also add column and index in separate commands.